### ISSUE TITLE
Can I apply ng-controller to the entire container at home.html#L2?

### ISSUE THREAD

**Issue by bwann52** [bwann52/Bloc-Chat#3]
https://github.com/bwann52/Bloc-Chat/blob/list-messages/app/templates/home.html#L24

**Comment by bdougie** [comment]
TLDR; Yes


Before answering this question I want to point out the structure of this question does not provide enough details, adds some misdirection to what you are eventually asking. For future questions follow the SCQA pattern. 

**S**ituation - What are you trying to do/
**C**omplication - What is the thing holding you back from doing this, the problem
**Q**uestion - What is your question
**A**nswer - What is your best guess on why there is a complication.

You can set $stateParams.room_id, which you have commented out [here](https://github.com/bwann52/Bloc-Chat/blob/list-messages/app/scripts/controllers/HomeCtrl.js#L9).

You could also leverage [ngClick](https://docs.angularjs.org/api/ng/directive/ngClick) for this, but I wouldn't do any of this because you are already setting the current room in the url.
```
 <a href="/{{room.$id}}"><li>{{room.$value}}</li></a>
```
So instead of setActiveRoom you can make it getActiveRoom and follow the below pattern
```js
function HomeCtrl(Room, $stateParams, Message) {
  this.currentRoom = getActiveRoom()

  function getActiveRoom() {
    return $stateParams.room_id
  }
}
```
